src/tree/v2/BranchDataItemCache.ts

```diff
@@ -0,0 +1,25 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ResourceGroupsItem } from './ResourceGroupsItem';
+
+export class BranchDataItemCache {
+    private readonly branchItemToItemCache: Map<unknown, ResourceGroupsItem> = new Map();
```

Comment:
Not critical, but should we change their names to `branchItemToResourceItemCache` / `resourceItemToBranchItemCache`? Or something along those lines?

Reply:
Figured we might as well go all the way and name it `branchItemToResourceGroupsItemCache` 😄